style7.css

```diff
@@ -0,0 +1,122 @@
+body {
+    max-width: 1510px;
+    font-family:'Raleway', sans-serif ;
```

Comment:
Your fonts are not plugged in anywhere. Therefore everything is rendered in Areal